metadata-ingestion/src/datahub/ingestion/source/ldap.py

```diff
@@ -387,8 +392,10 @@ def build_corp_user_mce(
 
         manager_urn = f"urn:li:corpuser:{manager_ldap}" if manager_ldap else None
```

Comment:
I think this also needs to get updated to respect use_email_as_username, but otherwise looking good

Reply:
`make_manager_urn` now also leverages `use_email_as_username` flag to create urn ( email exists) , default is `manager_ldap` .